ignite/contrib/handlers/param_scheduler.py

```diff
@@ -589,6 +589,7 @@ def simulate_values(  # type: ignore[override]
             durations: list of number of events that lasts a parameter scheduler from schedulers.
             param_names: parameter name or list of parameter names to simulate values.
                 By default, the first scheduler's parameter name is taken.
+            kwargs:
```

Comment:
this is a good question to what corresponds those kwargs

Reply:
Yes, `kwargs` is used in
```py
scheduler = cls(schedulers=schedulers, save_history=False, durations=durations, **kwargs)
```
`cls` is `ConcatScheduler` which only has 3 arguments, so kwargs is unnecessary?

Reply:
Yes, seems like that. Let's remove it from docstring and the function signature